s3-source-connector/src/main/java/io/aiven/kafka/connect/s3/source/S3SourceTask.java

```diff
@@ -16,19 +16,14 @@
 
 package io.aiven.kafka.connect.s3.source;
 
-import static io.aiven.kafka.connect.s3.source.config.S3SourceConfig.AWS_S3_BUCKET_NAME_CONFIG;
-import static io.aiven.kafka.connect.s3.source.config.S3SourceConfig.MAX_POLL_RECORDS;
-
 import java.lang.reflect.InvocationTargetException;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
I thought PMD would complain about importing the entire package, but it actually doesn't (At least locally) however it does have a number of complaints. Mostly setting variables as final and a few unused imports and variables.

./gradlew :s3-source-connector:pmdMain